examples/complete/variables.tf

```diff
@@ -1,4 +1,29 @@
-variable "example" {
+variable "region" {
   type        = string
-  description = "The value which will be passed to the example module"
+  description = "The AWS region in which to provision resources"
 }
+
+variable "availability_zones" {
+  type        = list(string)
+  default     = []
+  description = "List of availability zones in which to provision VPC subnets"
+}
+
+variable "kubernetes_version" {
+  type        = string
+  default     = "1.20"
```

Comment:
is `1.20` a real k8s version?

Reply:
not for EKS, but maybe for Spot?

Reply:
It will be soon. Our documentation says that leaving it empty will pick the latest Kubernetes version, but actually we have it set to default at 1.15. So I wanted to put in a better fake default for the test.